### ISSUE TITLE
Build with SVG support?

### ISSUE THREAD

**Issue by joelmccracken** [caldwell/build-emacs#37]
I was just trying to edit and preview a svg file, and it didn't
work. I did a little digging, and it seems like this version of emacs
is compiled without SVG.
- Is this true, or am I misunderstanding something?
- Could we compile with SVG?

**Comment by zbeekman** [comment]
:+1:

**Comment by skuro** [comment]
:+1:

**Comment by nloveladyallen** [comment]
This issue appears to be fixed, at least for me. Emacs 25.1, macOS 10.12.0.

**Comment by futurist** [comment]
Not fixed on my side: macOS 10.9.5, Emacs 25.1.1

After  `brew install librsvg`, the problem still there.

So I think it's the building problem.

**Comment by nloveladyallen** [comment]
@futurist Do you have ImageMagick installed? And did you download from the site, or build yourself?

**Comment by futurist** [comment]
@nloveladyallen 
Don't have ImageMagick installed
download from the site

**Comment by nloveladyallen** [comment]
@futurist It should work by just building from source after installing librsvg. ImageMagick adds support for lots of additional image types, if you care about anything other than the most common ones, but shouldn't be required for SVG support.

**Comment by caldwell** [comment]
@nloveladyallen, this bug is for building the Emacs binaries on https://emacsformacos.com with SVG support, not just Emacs in general. SVG is not included there quite yet.

**Comment by wyuenho** [comment]
This would be nice.

**Comment by mikehenry-se** [comment]
Yes, out-of-the-box SVG support in Emacs would certainly enhance the user experience.
Even MacPorts provides this.. ;-)

**Comment by stardiviner** [comment]
I'm using M1 Mac Mini, macOS 11.3

Emacs for macOS version: GNU Emacs 27.2 (build 1, arm-apple-darwin20.3.0, NS appkit-2022.30 Version 11.2.3 (Build 20D91)) of 2021-03-28

I found the new company-mode added built-in icons support for candidates with svg. But this Emacs version does not support svg image type.

I installed `libsvg`, `librsvg`, and `imagemagick` too. Don't work still.

**Comment by salutis** [comment]
+1 for SVG.

**Comment by stardiviner** [comment]
I checked the README, can't understand the build from source code steps. So I have to wait for maintainer to add support for SVG. That's hard for impatient me.

**Comment by hmelman** [comment]
Yes, please add librsvg to the release builds on https://emacsformacosx.com/.  At least the Emacs-28.1-3-universal.dmg does not seem to support them, though I think a previous Emacs 28 build did?
